docs/dev/test-architecture-s390x.adoc

```diff
@@ -0,0 +1,24 @@
+= test-architecture-s390x
+For all tests on s390x arch, the openshift cluster version is 4.3 or newer.
+The prerequisites, please refer to default https://github.com/openshift/odo/blob/master/docs/dev/test-architecture.adoc[test-architecture]!
+
+== Running integration and e2e tests locally
+
+Odo CI for s390x is running locally now.The scripts/configure-installer-tests-cluster-s390x.sh is available in the https://github.com/openshift/odo/tree/master/scripts[odo] repository to help configure the test environment and the scripts/openshiftci-presubmit-all-tests.sh is available in the https://github.com/openshift/odo/tree/master/scripts[odo] repository to run integration and e2e tests covered on s390x arch.  
+For running all covered integration and e2e tests on 4.x cluster, you can run:
+----
+	./scripts/openshiftci-presubmit-all-tests.sh
+----
+
+Similarly for running specific test on 4.x cluster, you can run:
+----
+	./scripts/configure-installer-tests-cluster-s390x.sh
+	make test-xxx-xxx
+----
+
+To generate the odo job file, run `make jobs` in https://github.com/openshift/release[openshift/release] for the odo pr and periodic tests.
```

Comment:
From line number 19 to end of the file i don,t think you need it. These lines are complete upstream related, So you can remove these lines.

Reply:
> From line number 19 to end of the file i don,t think you need it. These lines are complete upstream related, So you can remove these lines.

OK. I got it. Thanks.